### ISSUE TITLE
Question about alfred-pdf-tools and cpdf

### ISSUE THREAD

**Issue by gbstan** [xilopaint/alfred-pdf-tools#5]
I am using alfred-pdf-tools and find it really cool!

Just by curiosity, I was wondering if you had considered using Coherent Graphics PDF Tools:
https://github.com/coherentgraphics/cpdf-binaries
https://github.com/oncletom/homebrew-cpdf
http://community.coherentpdf.com

I am just curious to know whether it would make sense to use cpdf and how it compares with the libraries used by alfred-pdf-tools PyPDF2 and K2pdfopt

**Comment by xilopaint** [comment]
Given PyPDF2 is a pure Python library it allows much more customizations than CLIs like cpdf. Besides that PyPDF2 is much more lightweight than cpdf (434 KB against 4.3 MB). Anyway, if you'd like to see any new feature in Alfred PDF Tools you can ask here and if it can't be added with PyPDF2 or K2pdfopt you can also suggest a new library or CLI that allows it being implemented.

**Comment by gbstan** [comment]
Makes sense. Thank you for helping me understand the difference better. I don't have a specific use case that requires anything else for the moment. Alfred-PDF-Tools is really fantastic.
